applications/istio/bookinfo/bookinfo.yaml

```diff
@@ -0,0 +1,331 @@
+# Copyright 2017 Istio Authors
```

Comment:
The creation of native resources should not be needed, and should be componentized